saltyrtc/server/bin.py

```diff
@@ -246,3 +246,7 @@ def main():
     finally:
         if obj['logging_handler'] is not None:
             obj['logging_handler'].pop_application()
+
+
+if __name__ == '__main__':
```

Comment:
The line shouldn't be necessary as setuptools will generate a script that calls the `main` function above. Furthermore, running bin.py is probably not going to work as there are relative imports.